main.js

```diff
@@ -1,77 +1,79 @@
 'use strict';
-const { Telegraf } = require('telegraf');
+const Telegraf = require('telegraf');
 const Extra = require('telegraf/extra');
-const request = require('request');
-const bot = new Telegraf('1241797755:AAGIIpoXru5SWv4mDO5PhSD7N4G3x91mEBQ');
 
-const token = 'aa36752d4a3159859afd0e84b3abf7cacab10018';
+const FUNCTIONS = require('./modules/functions.js');
```

Comment:
```suggestion
const functions = require('./modules/functions.js');
```
Именование `camelCase`